addon/templates/components/mdc-grid-list/tiles/tile/primary.hbs

```diff
@@ -0,0 +1,3 @@
+{{yield (hash
+  primary-content=(component "mdc-grid-list/tiles/tile/primary/primary-content")
```

Comment:
This does miss out on the ability to "Using a div in place of an img" as specified in the MDC-Web docs. Perhaps it would be a good idea to yield a corresponding `primary-content-div` (or something like that)?

Reply:
It already supports doing this.  See the last example in the demo list.